source/transfers/contracts/ERC1155TransferHandler.sol

```diff
@@ -0,0 +1,22 @@
+pragma solidity 0.5.12;
+
+import "./interfaces/ITransferHandler.sol";
+import "./interfaces/IERC1155TransferHandler.sol";
+
+contract ERC1155TransferHandler is ITransferHandler {
+  function transferTokens(
+    address from,
+    address to,
+    uint256 param,
+    address token
+  ) external returns (bool) {
+    IERC1155TransferHandler(token).safeTransferFrom(
```

Comment:
I would say its misleading to name the interface `IERC1155TransferHandler` when this contract is `ERC1155TransferHandler`. To me that very strongly implies that the interface is the interface for this contract as thats the naming convention we have. e.g. ISwap is the interface for Swap.

I would say that you're casting an actual token, not a "handler". It would be better to be named `IERC1155Transfer`  or `IERC1155TokenTransfer` or something?

Reply:
removed IERC1155 and just used to interface that already exists in tokens.